### PR TITLE
Homepage update: Add sharing section, integrations list, and update links

### PR DESCRIPTION
**Pull request by domyen**
- Add sharing section that describes Storybook's sharing features
- Integrations list 
- Added/Updated links

Look at Chromatic for the full UI Review

image

### CODE REVIEW

File: src/components/screens/IndexScreen/IndexScreen.js [hunk 1]
```diff
@@ -240,14 +319,82 @@ export function PureIndexScreen({ ...props }) {
...
+          <BenefitItem
+            image={<img src="/images/home/share-import-stories.png" alt="Import stories" />}
+            title={
+              <>
+                <Code>import</Code> stories into other JavaScript tooling
+              </>
+            }
+            desc={
+              <>
+                Stories are a <Link href={`${docs}react/api/csf`}>portable standard</Link> based on
```
Comment (winkerVSbecks) [inline src/components/screens/IndexScreen/IndexScreen.js:380]: Would this be a better place to link to?

```suggestion
                Stories are a <Link href={`${docs}react/writing-tests/importing-stories-in-tests`}>portable standard</Link> based on
```

File: src/components/screens/IndexScreen/IndexScreen.js [hunk 2]
```diff
@@ -240,14 +319,82 @@ export function PureIndexScreen({ ...props }) {
...
+                ES6 modules. Write stories once and import them into any JavaScript library.
```
Comment (winkerVSbecks) [inline src/components/screens/IndexScreen/IndexScreen.js:381]: Is there a use case beyond testing? Right now this section doesn't convey "why" I would want to import stories anywhere. 

```suggestion
                ES6 modules. Write stories once and import them into any JavaScript testing library.
```

File: src/components/screens/IndexScreen/IndexScreen.js [hunk 3]
```diff
@@ -240,14 +319,82 @@ export function PureIndexScreen({ ...props }) {
...
+                online to give your team a universal reference point for feedback.
+              </>
+            }
+          />
+          <BenefitItem
+            image={<img src="/images/home/share-embed.png" alt="Embed stories" />}
+            title="Embed stories in your favorite sites"
+            desc={
+              <>
+                <Link href={`${docs}react/sharing/embed`}>Embed stories</Link> to showcase your work
```
Comment (winkerVSbecks) [inline src/components/screens/IndexScreen/IndexScreen.js:356]: This feels quite similar to the first point (publish to get feedback). How about:

> Embed stories to showcase live interactive examples in your docs. Works with Notion, Medium and any site supporting the oEmbed standard.

File: src/components/screens/IndexScreen/Integrations.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+const Wrapper = styled.div``;
+
+export default function CTA({ ...props }) {
+  return (
+    <Wrapper {...props}>
+      <Inner>
+        <Title>Integrations</Title>
+        <IntegrationsWrapper>
+          <IntegrationList>
```
Comment (winkerVSbecks) [inline src/components/screens/IndexScreen/Integrations.js:117]: Would love to turn all these into links (to addons or elsewhere). If you're worried about them getting stale then we can turn the whole section into a link to `/addons`

File: src/components/screens/IndexScreen/IndexScreen.js (hunk 3 above)
Comment (domyen) [reply src/components/screens/IndexScreen/IndexScreen.js:356]: 👍 sounds good

File: src/components/screens/IndexScreen/Integrations.js (hunk 4 above)
Comment (domyen) [reply src/components/screens/IndexScreen/Integrations.js:117]: Yea that's fair, can you make the change?

**Review by winkerVSbecks — APPROVED** [review]

**Pull request merged**